package.json

```diff
@@ -4,8 +4,10 @@
   "description": "task1 for Node.js mentoring program",
   "main": "index.js",
   "scripts": {
-    "start": "babel-node index.js",
-    "test": "node"
+    "compile": "babel-node index.js",
+    "test": "node",
+    "start": "nodemon --exec npm run compile",
+    "build": "mkdir -p build && babel index.js --out-file build/index.js"
```

Comment:
Nice idea, but it does not work because you transpile only one file.

Reply:
I move source code to folder and compile the whole folder now, thx for note